.evergreen/config.yml

```diff
@@ -1889,6 +1948,10 @@ buildvariants:
     os:
       - ubuntu-18.04
     async-runtime: "*"
+    crypt-shared:
+      - "enabled"
+    tls-feature:
```

Comment:
does this mean we now only test serverless with openssl? seems probably fine (I guess we were doing the opposite before, only rustls)

Reply:
I don't actually recall why I was only testing against openssl; I've updated this to test against both.  Currently serverless instance creation is timing out on evergreen, but initial runs showed passes for all of it.